phpcs.xml.dist

```diff
@@ -3,6 +3,7 @@
     <file>bootstrap.php</file>
     <file>src/</file>
     <file>tests/</file>
+    <file>wordpress-stubs.php</file>
```

Comment:
Please do not check the stubs. Stubs are having a hard life.

Reply:
it's now just a bootstrap file like bootstrap.php :)

Reply:
In other words: I'm not able to format stubs.

Reply:
My mistake.